FieldtypeTextareas support

FerdiAgrio opened the pull request:
Minor addition to use FieldtypeTextareas

FerdiAgrio committed:
Support for FieldtypeTextareas

Minor addition to use FieldtypeTextareas as well

FerdiAgrio committed:
To early commit

Bugfix - using original variable won't work

adrianbj commented:
Thanks @FerdiAgrio - mostly looks good, but from some quick testing I think there are some issues when removing videos from the textareas subfields - seems like they are not being deleted from the filesystem so we are ending up with orphaned images and then "-n" appendices are getting messed up. I think you might need to add some additional logic here: https://github.com/FerdiAgrio/GetVideoThumbs/blob/master/ProcessGetVideoThumbs.module#L221

Also wondering about the `$vURLf` name - it's not specifically a video url field - it could be a textarea within the textareas field, so perhaps a more generic name would be more appropriate.

Thanks again!

FerdiAgrio committed:
Renamed variable, generic check for FieldTypes

Renamed variable $vURLf to $searchField, one declaration for compatible FieldTypes and check in the children of FieldtypeTextareas if they are compatible

FerdiAgrio commented:
Hi Adrian,

I've tested the commit again (also in a few new projects) but found no problems. Although I did make a few better changes again. Maybe you could check those out as well or please send me more details about what didn't work the first time.

Thanks in advance!

adrianbj commented:
Thanks - I have accepted your PR, but I did make some changes as well as fixing something I broke in the last commit which was actually the issue with automatic deleting of images when a video no longer exists.

Please let me know if you have any problems with the current version which is now updated in the modules directory.

FerdiAgrio commented:
Hi Adrian,

Thanks for making the corrections and merging my addition to it. Keep up the good work!

Ferdi
